web/models/audit_log.ex

```diff
@@ -0,0 +1,31 @@
+defmodule HexWeb.AuditLog do
+  use HexWeb.Web, :model
+
+  schema "audit_logs" do
+    belongs_to :actor, User
+    field :action, :string
+    field :extra_params, :map
```

Comment:
the way I was thinking about it was: params=base_params+extra_params,base_params being actor_id, action, inserted_at. I think it's fine either way; params would be ok here with that other 'opts' variable name you mentioned elsewhere

> On Feb 27, 2016, at 11:39 PM, Aleksei Magusev <notifications@github.com> wrote:
> 
> In web/models/audit_log.ex:
> 
> > @@ -0,0 +1,31 @@
> > +defmodule HexWeb.AuditLog do
> > +  use HexWeb.Web, :model
> > +
> > +  schema "audit_logs" do
> > +    belongs_to :actor, User
> > +    field :action, :string
> > +    field :extra_params, :map
> Can it be just :params?
> 
> —
> Reply to this email directly or view it on GitHub.
>